back/cmd/openbsrv/main.go

```diff
@@ -85,15 +107,15 @@ func run() error {
 		return err
 	}
 
-	m := newServerMgmt(gsrv, hsrv, fsrv)
+	m := newServerMgmt(log, gsrv, hsrv, fsrv)
 
 	sm.Set(func(s *sigmon.State) {
 		if err := m.stop(); err != nil {
-			fmt.Fprintln(os.Stderr, err)
+			log.Error(err.Error())
```

Comment:
This would be an app-level error. If we cannot halt the program cleanly, we should return an error to the main func and print to stderr.

Reply:
For now leave it as a direct print to os.Stderr and file an issue to sort out how we want errors to bubble back up to main.

Reply:
Issue #160 now covers the errors collection for the stop() function in main.go.